AnnoDesigner/Localization/Localization.cs

```diff
@@ -343,6 +344,7 @@ public static void Init(ICommons commons)
                         { "Delete" , "Löschen" },
                         { "Duplicate" , "Duplikat" },
                         { "RotateAll" , "Alle drehen" },
+                        { "DeleteHovered" , "Schwebeobjekt löschen" },
```

Comment:
Maybe "Objekt unter Mauszeiger löschen"? (english would be something like "Delete object under mouse cursor")
The name "Schwebeobjekt" is not really used/known in german.

Reply:
I think I prefer "Delete object under cursor" anyway - I'll update the english translation as well.